src/toast/toast.style.ts

```diff
@@ -35,58 +35,48 @@ export default css`
     font: inherit;
     gap: var(--tap-sys-spacing-3);
     width: 100%;
-    font-family: var(--tap-sys-font-family);
+    font-family: var(--tap-font-family, var(--tap-sys-font-family)), serif;
```

Comment:
remove `serif` please. event the fallback font should be defined in the tokens.

Reply:
done